cmd/lncli/main.go

```diff
@@ -177,6 +177,7 @@ func main() {
 		verifyMessageCommand,
 		feeReportCommand,
 		updateFeesCommand,
+		payInvoiceCommand,
```

Comment:
The placement of the payinvoice command probably should be right above (or below?) the sendpayment command in the list.

Reply:
@halseth I have moved the command below sendPayment along with other invoice commands. PTAL :).